mira/metamodel/templates.py

```diff
@@ -678,7 +685,7 @@ class TemplateModel(BaseModel):
     templates: List[SpecifiedTemplate] = Field(
         ..., description="A list of any child class of Templates"
     )
-    parameters: Mapping[str, float] = Field(default_factory=dict,
+    parameters: Mapping[str, Parameter] = Field(default_factory=dict,
```

Comment:
can you clarify here (maybe with the `description` keyword in the `Field` constructor) - the keys in this dictionary are totally local to the given template model, and usually originate from the colloquial names used in its construction?

In some examples, I would guess the key will be different than the name in the `Parameter` object if it gets grounded and renamed

Reply:
Yes, in principle they can be different - the key would correspond to the way the parameter appears in rate laws and is required for reverse lookups (e.g., to determine whether a parameter appears in a given rate law). Doing otherwise would require a bit more work on handling custom rate laws (e.g., modifying an existing rate law in case something in it gets renamed at a higher level).